### ISSUE TITLE
Homebrew install fails due to missing file ModuleInformation.cmake

### ISSUE THREAD

**Issue by Felerius** [cginternals/cpplocate#22]
It seems the `ModuleInformation.cmake` file removed in 77c1aa3 is still required somewhere in the installation process. The following is result of trying to install cpplocate using the homebrew formula in [cginternals/homebrew-cginternals](https://github.com/cginternals/homebrew-cginternals):
```bash
$ brew install --HEAD cpplocate
==> Installing cpplocate from cginternals/cginternals
==> Cloning https://github.com/cginternals/cpplocate.git
Cloning into '/Users/david/Library/Caches/Homebrew/cpplocate--git'...
remote: Counting objects: 425, done.
remote: Compressing objects: 100% (334/334), done.
remote: Total 425 (delta 113), reused 240 (delta 70), pack-reused 0
Receiving objects: 100% (425/425), 1.11 MiB | 774.00 KiB/s, done.
Resolving deltas: 100% (113/113), done.
==> Checking out branch master
==> cmake . -DCMAKE_C_FLAGS_RELEASE=-DNDEBUG -DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG -DCMAKE_INSTALL_PREFIX=/usr/local/Cellar/cpplocate/HEAD
==> cmake --build . --target install
Last 15 lines from /Users/david/Library/Logs/Homebrew/cpplocate/02.cmake:
gmake[1]: Leaving directory '/private/tmp/cpplocate-20180423-35523-10ktg6d'
Install the project...
/usr/local/Cellar/cmake/3.11.1/bin/cmake -P cmake_install.cmake
-- Install configuration: "Release"
-- Installing: /usr/local/Cellar/cpplocate/HEAD-801246f/./VERSION
-- Installing: /usr/local/Cellar/cpplocate/HEAD-801246f/./cpplocate-config.cmake
-- Installing: /usr/local/Cellar/cpplocate/HEAD-801246f/./AUTHORS
-- Installing: /usr/local/Cellar/cpplocate/HEAD-801246f/./LICENSE
-- Installing: /usr/local/Cellar/cpplocate/HEAD-801246f/./README.md
CMake Error at cmake_install.cmake:56 (file):
  file INSTALL cannot find
  "/tmp/cpplocate-20180423-35523-10ktg6d/cmake/ModuleInformation.cmake".


gmake: *** [Makefile:89: install] Error 1

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/cginternals/homebrew-cginternals/issues
```

**Comment by scheibel** [comment]
Thanks for reporting. We'll have a look at it.

**Comment by scheibel** [comment]
Fixed with 377e0d1244d28b1c04822dd0d7f3e54942ab1fda.
